k8s/main.go

```diff
@@ -204,22 +200,13 @@ func main() {
 	}
 
 	ctx := ctrl.SetupSignalHandler()
-	ctrlConfig := configv1.ProjectConfig{}
+
 	options := ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "33405cb8.symphony",
 	}
-	if configFile != "" {
```

Comment:
What is the new requirement of using controller config file in newer version controller-runtime?

Reply:
We need to implement our own config reader if needed. Reference: https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/config/v1alpha1